README.md

```diff
@@ -130,13 +137,50 @@ posts for further information
     </td><td><code>""</code></td></tr>
 
   <tr><td>esHeapSize</td><td>integer</td>
-    <td>The size, in megabytes, of memory to allocate on each Elasticsearch node for the JVM heap. If unspecified, 50% of the available memory will be allocated to Elasticsearch heap, up to a maximum of 31744MB (~32GB). 
-    Take a look at <a href="https://www.elastic.co/guide/en/elasticsearch/reference/current/heap-size.html" target="_blank">the Elasticsearch documentation</a> for more information.  <strong>This is an expert level feature - setting a heap size larger than available memory on the Elasticsearch VM SKU will fail the deployment.</strong>
+    <td>The size, <em>in megabytes</em>, of memory to allocate on each Elasticsearch node for the JVM heap. If unspecified, 50% of the available memory will be allocated to Elasticsearch heap, up to a maximum of 31744MB (~32GB). 
+    Take a look at <a href="https://www.elastic.co/guide/en/elasticsearch/reference/current/heap-size.html" target="_blank">the Elasticsearch documentation</a> for more information.  <strong>This is an expert level feature - setting a heap size too low, or larger than available memory on the Elasticsearch VM SKU will fail the deployment.</strong>
     </td><td><code>0</code></td></tr>
 
+  <tr><td>esHttpSecurity</td><td>string</td>
+    <td>Either <code>Yes</code> or <code>No</code> to configure SSL/TLS for the HTTP layer of Elasticsearch. <strong>X-Pack plugin must be installed and either <code>esHttpCertBlob</code> or <code>esHttpCaCertBlob</code> provided</strong></strong>
+    </td><td><code>No</code></td></tr>
+
+  <tr><td>esHttpCertBlob</td><td>string</td>
```

Comment:
Maybe archive instead of blob for the rest of this ?

Reply:
I'd like to stick to using the parameter names for this. The `*Blob` suffix has been used in the template for encoded parameters, so would like to stick to this for consistency.

Reply:
Re-read this comment. I think renaming the PKCS#12 archive parameters would make sense e.g.

`esHttpCaCertBlob` -> `esHttpCaArchiveBlob`

_However_, the Application Gateway parameter for providing a PKCS#12 archive that has been in the template for over a year is called `appGatewayCertBlob`, so think `*CertBlob` would make for a more consistent naming approach, even though the parameter name is less/not technically correct.